Source: maxice8, m3tav3rse
Upstream: not upstreamable
Reason: Fixes building of package.

ERROR: Error in gtkdoc helper script:
ERROR: ['gtkdoc-mkhtml', '--path=/builddir/igt-gpu-tools-1.23/docs/reference/igt-gpu-tools:/builddir/igt-gpu-tools-1.23/build/docs/reference/igt-gpu-tools', 'igt-gpu-tools', '../igt-gpu-tools-docs.xml'] failed with status 5
I/O error : Attempt to load network entity http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl
warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl"
compilation error: file /usr/share/gtk-doc/data/gtk-doc.xsl line 11 element import
xsl:import : unable to load http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl

--- meson.build
+++ meson.build
@@ -232,14 +232,14 @@
 subdir('man')

 docs_info = 'No'
-if _build_docs
-       if _build_tests
-               subdir('docs')
-               docs_info = 'Yes'
-       elif _docs_required
-               error('Documentation requires building tests')
-       endif
-endif
+#if _build_docs
+#      if _build_tests
+#              subdir('docs')
+#              docs_info = 'Yes'
+#      elif _docs_required
+#              error('Documentation requires building tests')
+#      endif
+#endif
 build_info += 'Build documentation: ' + docs_info

 message('Build options')
